apps/src/lib/levelbuilder/script-editor/ScriptEditor.jsx

```diff
@@ -277,7 +281,11 @@ class ScriptEditor extends React.Component {
           const lessonGroups = mapLessonGroupDataForEditor(data.lesson_groups);
 
           this.props.init(lessonGroups, this.props.levelKeyList);
-          this.setState({lastSaved: data.updatedAt, isSaving: false});
+          this.setState({
+            lastSaved: data.updatedAt,
+            isSaving: false,
+            oldScriptText: data.lessonLevelData
```

Comment:
Have to update the script text after saving because if you stay on the page without reloading and without updating it would think there was a mismatch in the information between editor and database.